azkaban-common/src/test/java/azkaban/utils/EmailerTest.java

```diff
@@ -67,16 +67,18 @@ public void setUp() throws Exception {
     }
 
 
-
-
     /**
-     * test send email
+     * (test send email)
+     * if you want to run this case,
+     * please make sure these variable{host,mailPort,password,receiveAddr} effective.
```

Comment:
Do you mean
please make sure these variable{host,mailPort,password,receiveAddr} are set to real values that would allow an email to be sent successfully?